packages/kbn-grid-layout/grid/grid_height_smoother.tsx

```diff
@@ -42,26 +42,17 @@ export const GridHeightSmoother = ({
         dimensions.height ?? 0,
         smoothHeightRef.current.getBoundingClientRect().height
       )}px`;
+      smoothHeightRef.current.style.userSelect = 'none';
```

Comment:
This prevents stuff (like text in a markdown panel) from being highlighted as you drag another panel over it - we are temporarily disabling `userSelect` on the entire grid wrapper while the user drags.